#! /bin/sh /usr/share/dpatch/dpatch-run
## fix-version-2.1.3.dpatch by Diego Iastrubni <diego.iastrubni@xorcom.com>
##
## DP: Fix tarball contains a bug which identifies this verison as 2.1.2 and not
## DP: 2.1.3. This fixes it, and also makes an update for installed versions.

@DPATCH@
diff -urNad freepbx-2.1.3.dfsg/install_amp /tmp/dpep.2WVQnT/freepbx-2.1.3.dfsg/install_amp
--- freepbx-2.1.3.dfsg/install_amp	2006-11-26 12:30:45.000000000 +0200
+++ /tmp/dpep.2WVQnT/freepbx-2.1.3.dfsg/install_amp	2006-11-26 12:30:58.000000000 +0200
@@ -30,7 +30,8 @@
 		'2.1beta3',
 		'2.1.0',
 		'2.1.1',
-		'2.1.2'
+		'2.1.2',
+		'2.1.3'
 	);
 
 define("AMP_CONF", "/etc/amportal.conf");
diff -urNad freepbx-2.1.3.dfsg/SQL/newinstall.sql /tmp/dpep.2WVQnT/freepbx-2.1.3.dfsg/SQL/newinstall.sql
--- freepbx-2.1.3.dfsg/SQL/newinstall.sql	2006-09-26 01:43:11.000000000 +0300
+++ /tmp/dpep.2WVQnT/freepbx-2.1.3.dfsg/SQL/newinstall.sql	2006-11-26 12:30:45.000000000 +0200
@@ -64,7 +64,7 @@
 
 /*!40000 ALTER TABLE `admin` DISABLE KEYS */;
 LOCK TABLES `admin` WRITE;
-INSERT INTO `admin` VALUES ('need_reload','true'),('version','2.1.2');
+INSERT INTO `admin` VALUES ('need_reload','true'),('version','2.1.3');
 UNLOCK TABLES;
 /*!40000 ALTER TABLE `admin` ENABLE KEYS */;
 
diff -urNad freepbx-2.1.3.dfsg/SQL/newinstall.sqlite.sql /tmp/dpep.2WVQnT/freepbx-2.1.3.dfsg/SQL/newinstall.sqlite.sql
--- freepbx-2.1.3.dfsg/SQL/newinstall.sqlite.sql	2006-04-26 23:52:54.000000000 +0300
+++ /tmp/dpep.2WVQnT/freepbx-2.1.3.dfsg/SQL/newinstall.sqlite.sql	2006-11-26 12:30:45.000000000 +0200
@@ -27,7 +27,7 @@
 -- 
 
 INSERT INTO admin VALUES ('need_reload', 'false');
-INSERT INTO admin VALUES ('version','2.0.0');
+INSERT INTO admin VALUES ('version','2.1.3');
 -- --------------------------------------------------------
 
 -- 
diff -urNad freepbx-2.1.3.dfsg/upgrades/2.1.3/fix-version.sql /tmp/dpep.2WVQnT/freepbx-2.1.3.dfsg/upgrades/2.1.3/fix-version.sql
--- freepbx-2.1.3.dfsg/upgrades/2.1.3/fix-version.sql	1970-01-01 02:00:00.000000000 +0200
+++ /tmp/dpep.2WVQnT/freepbx-2.1.3.dfsg/upgrades/2.1.3/fix-version.sql	2006-11-26 12:30:45.000000000 +0200
@@ -0,0 +1 @@
+UPDATE admin SET value = '2.1.3' WHERE variable = 'version'
